src/Helpers/Constants.cs

```diff
@@ -55,6 +55,7 @@ public class Constants
 
     // Crons & Jobs
     public static readonly string MONITOR_WITHDRAWALS_CRON = "10 0/5 * * * ?";
+    public static readonly string MONITOR_CHANNELS_CRON = "0 0 0 * * ?";
```

Comment:
That's all nights at 00:00AM, shoudn't this be more periodic? like every hour?

Reply:
Hmm, I would do it more often

Reply:
moving to every hour